proto/regen/bond/v1alpha1/tx.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+package regen.bond.v1alpha1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/regen-network/regen-ledger/x/bond";
+
+// Msg defines the Msg service.
+service Msg {
+
+  // IssueBond issues a new bond
+  rpc IssueBond(MsgIssueBond) returns (MsgIssueBondResponse);
+
+  // SellBond sells a bond to a new owner
+  rpc SellBond(MsgSellBond) returns (MsgSellBondResponse);
+}
+
+// MsgIssueBond is the Msg/IssueBond request type.
+message MsgIssueBond {
```

Comment:
Related to https://github.com/regen-network/regen-ledger/discussions/815#discussioncomment-2305205,
Let's consider the following alternative for this RPC:
* `MsgNewBondEmission` - will create new unique denom D and N bonds with denom D and precision = 0 (exponent = 0). It will also create Emission object, storing the bond value V, face currency, maturity, interest rate etc... 
* Instead of `MsgIssueBond` we can do `MsgPrimaryBondSell` -> selling X bond form given emission to the borrower. It will fail if the amount of bonds left is smaller than X. Borrower has to borrow `X*V` face currency, to the issuer.